{"uuid": "acd17e7c-eaca-4473-952f-cc77e27ac3b2", "children": ["512d37cb-df86-475d-afd3-e42dd5b6cd43"], "befores": [{"name": "get_test_user_token_fixture", "status": "broken", "statusDetails": {"message": "KeyError: 'data'\n", "trace": "  File \"C:\\Users\\changan\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\changan\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"C:\\Users\\changan\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\changan\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\changan\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 917, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\changan\\Documents\\python\\RbacTest\\testcases\\conftest.py\", line 67, in get_test_user_token_fixture\n    return loginInfo.data['data']['token']\n"}, "steps": [{"name": "前置步骤 ==>> 获取测试用户token", "status": "passed", "start": 1730794775433, "stop": 1730794775433}], "start": 1730794775433, "stop": 1730794775849}], "start": 1730794775433, "stop": 1730794775916}